main.go

```diff
@@ -157,6 +158,28 @@ func main() {
 			total["events"].(float64), index["total"].(float64), tput["throughput"].(float64), inputs["total"].(float64), elapsed), nil)
 	}
 
+	if strings.HasSuffix((*indexcrit), "%") && strings.HasSuffix((*indexcrit), "%") {
```

Comment:
Isn't this supposed to be indexwarn and indexcrit?
```suggestion
	if strings.HasSuffix((*indexwarn), "%") && strings.HasSuffix((*indexcrit), "%") {
```